### DISCUSSION TITLE
How do you make a file an MP3?

### DISCUSSION THREAD

**Discussion by PNLP918** [JunkFood02/Seal#710] [Q&A] [answered]
Title, also i do know that I could just use a website to convert them, but i rather just use the feature
(Also only reason I want the .mp3 is so that it works with my music player)

**Comment by Marco-9456** [comment, marked as answer]
>


https://user-images.githubusercontent.com/41828058/232324286-31e19a5f-5060-4e08-ba4f-566434660e38.mp4

**Reply by PNLP918** [reply]
It didn't work, but it still played in my music player so guess it works maybe
